### PR TITLE
Fix color contrast issue

### PR DESCRIPTION
**Pull request by t-rufang**
- This is to fix the color contrast accessibility issue at [#1852631](https://dev.azure.com/mseng/VSJava/_workitems/edit/1852631)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JLabelWithTheme.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.intellij.ui
+
+import com.microsoft.azure.hdinsight.common.DarkThemeManager
+import org.apache.commons.lang3.StringUtils
+import javax.swing.JLabel
+
+class JLabelWithTheme(text: String = ""): JLabel(text) {
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JLabelWithTheme.kt]: Might the name ‘ErrorLabel’ be more sense?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JTextAreaWithTheme.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.intellij.ui
+
+import com.microsoft.azure.hdinsight.common.DarkThemeManager
+import org.apache.commons.lang3.StringUtils
+import javax.swing.JTextArea
+
+class JTextAreaWithTheme : JTextArea() {
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JTextAreaWithTheme.kt]: Might the name ‘ErrorTextArea’ be more sense?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JLabelWithTheme.kt (hunk 1 above)
Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JLabelWithTheme.kt]: Fixed in new iteration

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JTextAreaWithTheme.kt (hunk 2 above)
Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JTextAreaWithTheme.kt]: Fixed in new iteration

**Review by wezhang — APPROVED** [review]
LGTM! Thanks Rui!

**Pull request merged**